### PR TITLE
docs: update README to contain build & test instructions

### PR DESCRIPTION
**Pull request by metaproph3t**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,47 @@
...
+Before you can build the program, you will first need to install the following:
+- [Rust](https://www.rust-lang.org/tools/install)
+- [Solana](https://docs.solana.com/cli/install-solana-cli-tools)
+- [Anchor](https://www.anchor-lang.com/docs/installation) (v0.27.0)
 
-### Submodules
+### Building
 
-After cloning this repo you'll need to init and update its git submodules.
-Consider setting the git option `submodule.recurse=true`.
```
Comment (metaproph3t) [inline README.md:32]: I noticed that there are currently no active submodules (unless I did something wrong) - is this planned? If so, I can add these lines back

Comment (binyebarwe) [reply README.md:32]: That's right, was my bad. I've just fixed it here https://github.com/openbook-dex/openbook-v2/pull/7

Comment (binyebarwe) [reply README.md:32]: So please add back these lines and I'll merge this PR

Comment (metaproph3t) [reply README.md:32]: Sure thing, sir

**Pull request merged**